engine/flink/avro-util/src/main/scala/pl/touk/nussknacker/engine/avro/fixed/FixedKafkaAvroSchemaProvider.scala

```diff
@@ -0,0 +1,57 @@
+package pl.touk.nussknacker.engine.avro.fixed
+
+import io.confluent.kafka.schemaregistry.client.MockSchemaRegistryClient
+import org.apache.avro.Schema
+import org.apache.flink.api.common.typeinfo.TypeInformation
+import org.apache.flink.streaming.connectors.kafka.{KafkaDeserializationSchema, KafkaSerializationSchema}
+import pl.touk.nussknacker.engine.api.typed.typing
+import pl.touk.nussknacker.engine.avro.schemaregistry.SchemaRegistryClient
+import pl.touk.nussknacker.engine.avro.schemaregistry.confluent.ConfluentSchemaRegistryClientFactory.ConfluentSchemaRegistryClient
+import pl.touk.nussknacker.engine.avro.schemaregistry.confluent.formatter.ConfluentAvroToJsonFormatter
+import pl.touk.nussknacker.engine.avro.schemaregistry.confluent.{ConfluentAvroDeserializationSchemaFactory, ConfluentAvroSerializationSchemaFactory, ConfluentSchemaRegistryClientFactory}
+import pl.touk.nussknacker.engine.avro.typed.AvroSchemaTypeDefinitionExtractor
+import pl.touk.nussknacker.engine.avro.{AvroUtils, KafkaAvroSchemaProvider}
+import pl.touk.nussknacker.engine.kafka.{KafkaConfig, RecordFormatter}
+
+/**
+  * This class has fake implementation of serializer and deserializer based on Confluent implementations - it's hack
+  * to serialization / deserialization schema without SchemaRegistry.
+  *
+  * @TODO: In future we should create own serializator and deserializator for FixedSchema
```

Comment:
I don't think I understand this comment. In the future we may do some refactoring to get rid of RecordFormatter or sth, but we will still have logic based on Confluent implementation of SchemaRegistry?

Reply:
I changed a litle this comment - please check it!